xml/depl_multiple_networks.xml

```diff
@@ -0,0 +1,433 @@
+ start=192.168.135.2,end=192.168.135.127 --gateway 192.168.135.1 public2 \
+ 192.168.135.0/24 --enable_dhcp False
+</screen>
+</step>
+
+<step>
+    <para>
+  The next step is to create a second router:  
+</para>
+<screen>
+sudo neutron router-create router2
+</screen>
+</step>
+
+<step>
+    <para>
+ Connect router2 to the second external network:       
+</para>
+<screen>
+sudo neutron router-gateway-set router2 public2
+</screen>
+</step>
+
+<step>
+    <para>
+        Create another private network and connect it to router2:
+</para>
+<screen>
+sudo neutron net-create priv-net
+sudo neutron subnet-create priv-net --gateway 10.10.10.1 10.10.10.0/24 \
+ --name priv-net-sub
+sudo neutron router-interface-add router2 priv-net-sub
+</screen>
+</step>
+
+<step>
+    <para>
+ Boot a VM setting the web security group and priv-net-sub network.       
+</para>
+</step>
+
+<step>
+    <para>
+ Assign a floating IP address to the VM, this time from network public2.
+</para>
+</step>
+
+<step>
+    <para>
+        From the node verify that SSH is working by opening an SSH session to 
+the VM.
+</para>
+</step>
+</procedure> 
+</sect1>
+
+<sect1 xml:id="sec.automatenet">
+    <title>Automating Network Setup</title>
+    <para>
+        Ideally, we need to be able to setup a second external network using 
+&crow;. This can be done as manual step, but this is error-prone.
+</para>
+
+<para>
+We should be able to handle the configuration file of the OVS agent that is on 
+the network node in a separate way. The OVS agent runs on every compute node, 
+and with the current configuration the compute node is not connected to the 
+external network.
+</para>
+<para>
+    If the parameter <parameter>bridge_mappings</parameter> is set, and the 
+agent doesn't find the bridges specified in the mappings, it will exit. That is
+why we need one configuration file for the agent running on the network node 
+with <parameter>bridge_mappings</parameter> specified, and one configuration 
+for the agent running on the compute node without 
+<parameter>bridge_mappings</parameter> . (Unless you embrace the 
+&o_netw; Distributed Virtual Router (DVR) and set a connection to the 
+external network on every compute node.)
+</para>
+<tip>
+    <title>Setting up DVR</title>
+    <para>
+For setting up DVR apart from having a connection to the external network 
+on every compute node, it seems that only configuration changes are required, 
+and the support of l2 POP and VXLAN.
+</para>
+</tip>
+
+<para>
+To automate creating external networks you need  
+<link xlink:href="https://github.com/SUSE-Cloud/automation">
+    <citetitle>mkcloud</citetitle></link>
+Start by following these steps to modify the &o_netw; &barcl;.
+</para>
+
+<procedure xml:id="pro.automate-net">
+    <step>
+    <para>
+        First, run the following script.
+    </para>
+    <screen>
+sudo ./runtest.c18.rossella_s prepare setupadmin addupdaterepo runupdate \ 
```

Comment:
So I need to rewrite the whole thing and delete most of it :) Good enough, thank you @rossella!